src/cth_readable_compact_shell.erl

```diff
@@ -119,20 +126,24 @@ on_tc_fail(TC, Reason, State=#state{suite=Suite, groups=Groups}) ->
 
 %% @doc Called when a test case is skipped by either user action
 %% or due to an init function failing. (>= 19.3)
-on_tc_skip(Suite, {TC,_Group}, Reason, State=#state{groups=Groups}) ->
-    ?SKIP(Suite, "~s", [format_path(TC,Groups)], Reason),
+on_tc_skip(Suite, {TC,_Group}, Reason, State=#state{groups=Groups, opts=Opts}) ->
+    skip(Suite, TC, Groups, Reason, Opts),
     State#state{suite=Suite};
-on_tc_skip(Suite, TC, Reason, State=#state{groups=Groups}) ->
-    ?SKIP(Suite, "~s", [format_path(TC,Groups)], Reason),
+on_tc_skip(Suite, TC, Reason, State=#state{groups=Groups, opts=Opts}) ->
+    skip(Suite, TC, Groups, Reason, Opts),
     State#state{suite=Suite}.
 
+skip(Suite, TC, Groups, Reason, Opts) ->
+    Verbose = proplists:get_value(verbose, Opts),
+    ?SKIP(Suite, "~s", [format_path(TC,Groups)], Reason, Verbose).
+
 %% @doc Called when a test case is skipped by either user action
 %% or due to an init function failing. (Pre-19.3)
 on_tc_skip({TC,Group}, Reason, State=#state{suite=Suite}) ->
-    ?SKIP(Suite, "~p (group ~p)", [TC, Group], Reason),
+    ?SKIP(Suite, "~p (group ~p)", [TC, Group], Reason, true),
     State;
 on_tc_skip(TC, Reason, State=#state{suite=Suite}) ->
-    ?SKIP(Suite, "~p", [TC], Reason),
+    ?SKIP(Suite, "~p", [TC], Reason, true),
```

Comment:
since this is the compact shell one, should verbose default to `false` instead of `true` here?

Reply:
I might've read this wrong initially. I was under the impression that

```erlang
%% @doc Called when a test case is skipped by either user action
%% or due to an init function failing. (Pre-19.3)
```

applied to both function clauses. I didn't test with Pre-19.3, but wanted to keep behaviour there, as-was. If you want I can change it, but not sure you still wanna support Pre-19.3. The same is true for the other shell.

Otoh, on `on_tc_skip` for `>= 19.3`, we read `verbose` from `Opts` which might throw (as per the implementation I propose, since I don't propose a default).

So, two separate discussions, for me:

1. do you want to support pre-19.3?
2. what default to you want for the `verbose` option? `true` or `false`?

Reply:
I mostly went from the file name. I was assuming the compact shell one was meant to be more compact by default than the readable shell but now that I double-checked, the compact shell _is_ the default so it's probably fine as-is.

Reply:
1. There's no explicit need to support pre-19.3 anymore no. I think people can use the old builds for that anyway. You don't have to go out of your way to drop support though
2. I'd keep the default the same behavior as we have right now, and let people override it. If we want to change the default in rebar3 we can also start tweaking the options we pass when setting up the hooks. Just keeps compatibility longer.

Reply:
👍 the changes I propose, then, should conform to all expectations. Verbose is `true` by default in this pull request (and backwards compatibility is hopefully guaranteed), and this option will eventually be exposed to `rebar3` either with the same default (or otherwise, if you prefer).

Reply:
yep, I'll check out the branch and try it before merging & publishing